nemo_skills/dataset/bfcl_v4/prepare.py

```
@@ -101,9 +104,14 @@ def process_file(repo_root_dir, input_file, output_file, model_type="llama-nemot
             if idx == 0:
                 LOG.info(f"Processing {test_category}")
 
-            # TODO: Current preprocessing can be model dependent. This could be moved to inference time as well
-            # Convert class-based method calls to function calls
-            instance = process_multi_turn_test_case(instance, repo_root_dir)
+
+                test_cases_to_generate = clean_up_memory_prereq_entries(test_cases_to_generate)
```

Ruff (0.13.3)

Undefined name `test_cases_to_generate`

(F821)